esbuild-node-externals/README.md

```diff
@@ -78,6 +78,10 @@ Make package.json `peerDependencies` external.
 
 Make package.json `optionalDependencies` external.
 
+#### `options.allowList` (default to `[]`)
+
+Specify packages which are not marked as external.
```

Comment:
Let's add a sentence to make it really clear. `They will be included in the bundle`

Reply:
Good point, updated